docs/get-started/deployment-and-usage.md

```diff
@@ -99,6 +99,51 @@ this approach does not guarantee running once on every node. For example,
 tainted, non-ready nodes or some other reasons in Job scheduling may cause some
 node(s) will run extra job instance(s) to satisfy the request.
 
+### Helm Chart
+
+This chart bootstraps a Node Feature deployment on a Kubernetes cluster using the Helm package manager.
+
+#### Prerequisites
+
+[Helm package manager](https://helm.sh/) should be installed.
+
+#### Installing the Chart
+
+To install the chart with the release name network-operator:
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery/
+cd node-feature-discovery/deployment
+helm install network-operator ./network-operator/
```

Comment:
> helm install network-operator ./network-operator/

helm install node-feature-discovery ./node-feature-discovery/

IMO you should replace below as well  (network-operator -> node-feature-discovery)